web/src/testUtils.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
Could you rename this to `test-utils.tsx` and move it into the `tests` folder where we already have other files that are only concerned with test setup?